### DISCUSSION TITLE
Influencing the spatial distribution of a Phase

### DISCUSSION THREAD

**Discussion by hem-28** [neperfepx/neper#314] [Q&A] [answered]
In cold rolled multiphase steels, one of the phase accumulates to form a banded structure parallel to the rolling direction (As seen in the image below)
As far as I understood, it is not possible to influence the spatial distribution of a phase. Would there be some tricks to achieve this in Neper?
![166463E2-7941-4BC2-9F1A-EAADAE75EFED](https://user-images.githubusercontent.com/89460629/139383326-742534fb-b493-4c78-a998-363021b233c8.jpeg)
Thanks!

**Comment by rquey** [comment, marked as answer]
Here is an example:
```
$ neper -T -n 5000 -group "((z>0.45&&z<0.55)||id<500)?1:2"
$ neper -V n5000-id1.tess -datacellcol group -cameraangle 18 -print img
```
![image](https://user-images.githubusercontent.com/2659376/139395539-9a63aedb-30e2-4abf-bae1-6fe05abcf295.png)

**Reply by yiyuxingchen** [reply]
1. Can the grains other than the red grains be given color according to the orientation or other colscheme？
2. Is it possible to output the color of each die in the V module？
Thanks in advance.
